### ISSUE TITLE
Little more in depth bank argument

### ISSUE THREAD

**Issue by WietseWind** [WietseWind/xrp-fud-bingo#3]
By /u/Apocrypton (Reddit)

I tried to trim it down as much as possible, it got a little longer than I was hoping. Not sure if this is the type of thing you're looking for, feel free to edit it as you see fit if you want to use it.

> One of the arguments used against XRP is that cryptocurrency is inherently against the current financial system, so any cryptocurrency that helps financial institutions is evil. On the surface this seems somewhat reasonable, but at its core cryptocurrency is about reducing the power that the major institutions hold over all of us. Many banks and institutions are customer oriented, and they are 100% necessary for a functioning economy. Even if cryptocurrency rules the world we will still need a banking system for many reasons, credit and lending being major ones. It’s the bad actors that have a stranglehold over the system that we should be focused on, not the concept of banks in general.

> For those unaware, Ripple's intended use for XRP is to convert fiat for cross border payments. It is meant to be a cheaper and faster option than the correspondent banking system. XRP can help to level the playing field.

> The current correspondent banking system is old, outdated, and benefits the banking giants of the world disproportionately. The largest banks act as hubs, they facilitate cross border transactions and reap the massive profit in the form of fees that come along with it. Your local bank and credit union does not benefit from correspondent banking, it is hurt by it. Their customers sending money cross border are hurt by it.

> The banks that stand to gain the most from XRP aren't JP Morgan Chase and Wells Fargo, the banks that can benefit from XRP are the small banks that are held hostage by the current system and forced to pay the institutions at the top every time they send a cross border payment.

> Reducing the reliance we all have on the banking giants of the world is one of the main goals of cryptocurrency. XRP has a realistic chance of helping this happen. Rather than charging massive fees to their customers that are passed on to correspondent banks, smaller institutions can offer lowered prices, increased speed, and ensure that more of every cross border payment lands in the pockets of those that need it most. This has the secondary effect of taking fees and power away from the institutions at the top.

> With Ripple's help, XRP can allow smaller financial entities to thrive and free them from the grips that the banking cartels have over them.

> With that being said, improving cross border transactions is just one part of what the XRP Ledger is capable of. Most people seem to be unaware that you can issue digital assets on the XRP Ledger. This can be useful for companies that want to issue tokens for everything from asset backed tokens to tradeable frequent flyer miles. There are almost an unimaginable amount of uses for this, many of which we probably haven't even thought of.

> There is also the fact that the XRP Ledger is a decentralized exchange in an of itself. People are more and more beginning to realize that DEX's need to be an option and the XRP Ledger is a great place to start. The Interledger Protocol that Ripple has developed allows for cross chain communication and is excellent technology to build decentralized exchanges on top of. There is already millions being traded per day over the XRP Ledger already in everything from BTC to DOGE.

> Then there are the other companies building use cases in the XRP ecosystem. For example, Coil was recently introduced as a company looking to build a micropayment system for the web. In this way we can pay content creators directly in very small increments rather than relying on ad revenue.

> Tl;dr XRP may benefit banks and FI's, but it benefits the ones at the bottom and takes away power and profit from those at the top. This is in line with the crypto ethos that many in our communities subscribe to. That’s also not its only use, as the ILP allows for cross chain communication and the XRP Ledger functions as a DEX. There are also other companies that are hard at work building a multitude of use cases for XRP and the XRP Ledger.
